config.yaml

```diff
@@ -125,3 +125,11 @@ options:
       One of `Admin`, `Alpha`, `Gamma`, `Public`, `sql_lab`
     type: string
     default: Public
+  http-proxy:
+    description: |
+      The HTTP_PROXY value.
+    type: string
+  https-proxy:
+    description: |
+      The HTTPS_PROXY value.
+    type: string
```

Comment:
Do we also need a `no-proxy` config?

Reply:
We don't need it for our current use cases and I'm not entirely sure if this is supported by Superset - but let me check.

Reply:
I have added a `no-proxy` value.